bigquery_etl/docs/derived_datasets/generate_derived_dataset_docs.py

```diff
@@ -122,8 +122,10 @@ def generate_derived_dataset_docs(out_dir, project_dir):
             if dataset_path.is_dir()
             and all(
                 suffix not in str(dataset_path)
-                for suffix in ConfigLoader.get(
-                    "default", "non_user_facing_dataset_suffixes", fallback=[]
+                for suffix in tuple(
```

Comment:
this conversion seems unnecessary given that the result is being directly iterated, which won't care whether it's a tuple or a list?

Reply:
right